<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome</class>
 <widget class="QWidget" name="Welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>E-Lock</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1221</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane{
	border: 2px solid rgb(210, 210, 210);
	background:rgb(246, 246, 246);
	border-top-color:transparent;
}
QTabWidget::tab-bar{
	background:rgb(0, 0, 0);
	subcontrol-position:center;
}
QTabBar::tab{
	width:300px;/*宽度根据实际需要进行调整*/
	height:100px;
	background:rgb(210, 210, 210);
	border: 2px solid rgb(210, 210, 210);
	border-top-left-radius: 8px;
	border-top-right-radius: 8px;
    font-size:30px;
}
QTabBar::tab:selected{	
	background:rgb(246, 246, 246);
	border-bottom-color:rgb(246, 246, 246);
}
QTabBar::tab:!selected{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(240, 240, 240, 255), stop:0.5 rgba(210, 210, 210, 255), stop:1 rgba(225, 225, 225, 255));
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Face Unlock</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>500</y>
       <width>641</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CLICK TO UNLOCK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lock_btn">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>180</y>
       <width>271</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>LOCK
NOW</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Code Unlock</string>
    </attribute>
    <widget class="QLineEdit" name="passwd_Edit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>641</width>
       <height>120</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>500</y>
       <width>641</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CLICK TO UNLOCK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Create</string>
    </attribute>
    <widget class="QLabel" name="label_save">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>500</y>
       <width>431</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="save_btn">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>10</y>
       <width>271</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SAVE
IMAGE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwd_Edit_new">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>250</y>
       <width>271</width>
       <height>120</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Change New Code</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_code_btn">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>380</y>
       <width>271</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SAVE
CODE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>500</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Delete</string>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>10</y>
       <width>271</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_delete">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>481</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_btn">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>400</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
